client/src/Components/Pans.js

```diff
@@ -1,15 +1,45 @@
-import React, { useState } from "react";
+import React, { useState, useContext } from "react";
+import { GlobalContext } from "../Contexts/GlobalContext";
 import EditImg from "../Assets/img/icons8-edit(1).svg";
 import ScheduleImg from "../Assets/img/schedule.svg";
+import ArchiveImg from "../Assets/img/archive.png";
+import RestoreImg from "../Assets/img/restore.png";
 import EditPopUp from "./EditTask/EditPopUp";
+
 const Pans = (props) => {
```

Comment:
It's generally a good idea to de-structure the props that are coming into a component. It allows us to see at first glance what props the component is expecting.